machine/webapi/clearml_nmt_engine_build_job.py

```diff
@@ -83,8 +83,11 @@ def run(self, check_canceled: Optional[Callable[[], None]] = None) -> None:
                     check_canceled()
                 _translate_batch(model, pi_batch, source_tokenizer, target_detokenizer, writer)
 
-        print("Saving NMT model")
-        self._nmt_model_factory.save_model()
+        if self._config.save_model:
```

Comment:
I am planning on removing this line of code, so this isn't necessary.

Reply:
I'll tear it out now and we can track the need to add it in here: #12.